Covid Patients/DetailsView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DetailsView.swift
+//  Covid Patients
+//
+//  Created by Yassen Joba on 05/07/2023.
+//
+
+import Cocoa
+import AppCommonComponents
+
+class DetailsView: NSView, NibLoadable {
+
+    var name: String = ""{
```

Comment:
try to do a setup function with patient model itself and inside it you can set whatever value 
this will allow for the future to add more info without needing to upgrade the data source itself